### DISCUSSION TITLE
Unicode character not supported?

### DISCUSSION THREAD

**Discussion by potmandev** [photopea/photopea#7364] [Q&A]
I've tried pasting the unicode character 䶸 into photopea and it doesn't pop up, I've tried using fonts like Noto Sans but it just doesnt pop up. Any help?

**Comment by photopea** [comment]
I just tried it and I can not find any font in our database which supports your character.

Have you been able to type this character with some font in some program? What font did you use? Did you try to load that font into Photopea?

**Reply by potmandev** [reply]
i tried to write it in paint.net, which it worked with calibri, but when i load calibri into photopea it doesnt support the character

**Reply by photopea** [reply]
Could you set it to a different font in Paint.net ? If yes, does the character look different? If it always looks the same, it is probably using some internal font as a substitute.
